OpenEXR/IlmImf/ImfMultiPartInputFile.cpp

```diff
@@ -573,8 +573,7 @@ MultiPartInputFile::Data::chunkOffsetReconstruction(OPENEXR_IMF_INTERNAL_NAMESPA
             
             if(partNumber<0 || partNumber>int(parts.size()))
```

Comment:
Whenever we encounter T() or (T) casts, we should replace them with static_cast<T>, it just looks more intentional.